docs/guides/admin/docs/modules/awss3archive.md

```diff
@@ -46,13 +46,33 @@ The Opencast AWS S3 Archive service configuration can be found in the
 Using S3 Archiving
 ------------------
 
-There are two major methods to access S3 archiving features: manually, and via a workflow.  Amazon S3 archiving is not
-part of the default workflows and manual S3 offload is disabled by default.  To enable manual S3 offload you must edit
-the `offload.xml` workflow configuration file and change `var s3Enabled = false;` to `var s3Enabled = true;`.  To
-manually offload a media package follow the directions in the user documentation.
+Moving assets to S3 is done via a workflow operation handler added as part of a workflow.  This can be part of your
+main workflow that runs on all of your recordings, or a small manually activated workflow accessed on-demand from the
+tasks menu.  The workflow operation handler definition looks like this
+```
+    <operation
+      id="move-storage"
+      description="Offloading to AWS S3">
+      <configurations>
+        <configuration key="target-storage">aws-s3</configuration>
+      </configurations>
+    </operation>
+```
+
+Assets in S3 continue to be accessible to Opencast, however there may be cases where you wish to restore your content
+back to your local storage.  This can be accomplished using the same workflow operation definition as above, and
+changing the `target-storage` configuration value from `aws-s3` to `local-filesystem` like so
+```
+    <operation
+      id="move-storage"
+      description="Restoring from AWS S3">
+      <configurations>
+        <configuration key="target-storage">local-filesystem</configuration>
+      </configurations>
+    </operation>
+```
```

Comment:
I'm unsure if this "restore paragraph" is a good advice without more context. "Moving" is always based on snapshots. The `target-version` is used to specify what snapshot to move. It not specified (as done here) all snapshots are moved. This may be what is desired, but not necessarily.

The issue is that new snapshots are *always* created locally. You, therefor, mention that `move-storage` could be part of all workflows. Instead, I suggest using the tiered storage feature that moves older snapshots automatically to S3, e.g., after two weeks. Especially during the time of the ingest, there could be quite a lot of new snapshots because of metadata/ACL changes or workflows. The problem is that shapshots always contain all selected data, resulting in many downloads from S3.

When you know that a certain workflow will create a new shapshot, I would also not use `move-storage` as this would restore existing snapshots. Instead I would run `ingest-download` at the start of the workflow, which will put S3 files into the workspace. The `snapshot` operation is then very quick, but I think some operations also assume that everything is available locally (at least, I remember having problems with `sanitize-adaptive` when I did not download everything beforehand).

Reply:
Do you by any chance have a workflow that's already set up to do that?  That makes a ton of sense.

Reply:
We still publish a reference to our workflows here (https://github.com/opencast/community-workflows/tree/master/13.x/educast.nrw), but they are completely different from upstream workflows and assume certain changes. However, the `ingest-download` workflow is rather simple: https://zivgitlab.uni-muenster.de/educast-nrw/opencast/workflows/-/blob/educast/13/src/workflows/educast-partial-setup-workspaces.yaml. This is included in every main workflow that will eventually create a snapshot or otherwise needs files locally.